distro/wildfly/modules/src/main/modules/org/camunda/bpm/camunda-engine/main/module.xml

```diff
@@ -30,7 +30,11 @@
     <module name="org.graalvm.regex.regex" services="import"/>
     <module name="org.graalvm.sdk.graal-sdk" services="import"/>
     <module name="org.graalvm.truffle.truffle-api" services="import"/>
+    <module name="org.camunda.feel.feel-engine" services="import"/>
+    <module name="org.camunda.bpm.dmn.camunda-engine-feel-scala" services="import" export="true" />
```

Comment:
❓ We can double check if export is really needed here:
```suggestion
    <module name="org.camunda.bpm.dmn.camunda-engine-feel-scala" services="import"/>
```

Reply:
Removed `export="true"` as it is not required.